ruby/tests/well_known_types_test.rb

```diff
@@ -13,10 +13,15 @@ def test_timestamp
     assert_equal Time.at(12345), ts.to_time
     assert_equal 12345, ts.to_i
 
-    ts.from_time(Time.at(123456, 654321))
+    time = Time.at(123456, 654321)
+    ts.from_time(time)
     assert_equal 123456, ts.seconds
     assert_equal 654321000, ts.nanos
-    assert_equal Time.at(123456.654321), ts.to_time
+    assert_equal time, ts.to_time
+
+    time = Time.now
```

Comment:
IIRC I simply wanted a timestamp more accurate than milliseconds without running in some other floating point errors (which one would encounter if using i.e. `Time.at(123456, 654321.321)`)

I'll update the PR with some other way to do that.